proposals/drafts/campaign_finance_filings.rst

```diff
@@ -0,0 +1,312 @@
+two is muddy, but this proposal tries to keep Filings as minimal as possible.
+
+Finally, the different entity types presented here present either current or
+likely future conflicts with other entities in the Open Civic Data namespace, so
+this proposal doesn't make any attempt to resolve those conflicts since they
+seem ancillary for now.
+
+TODO
+----
+* Make filing activity actions use same pattern as OCD bill actions (@aepton)
+
+Questions to answer
+-------------------
+* How to model Committee Status and Committee Purpose inside of OCD Organization
+  models?
+* How to reconcile multiple reports that describe the same contribution,
+  expenditure or other event?
+* How to handle different types of Committees beyond just imputing their types
+  based on (say) whether they are oriented toward more than one Candidate?
+* How should amendments work? Should there be some notion of versions of
+  filings? Should all data from a given filing, even if 99% is redundant with a
+  prior version of same filing, be stored, or should we store a diff?
+* Should we make/preserve the distinction FEC makes between contributions and
+  receipts (every contribution is a receipt; not every receipt is a
+  contribution)? How do we handle presumably-unique transaction IDs that
+  nevertheless have to get versioned somehow?
+
+Implementation
+==============
+
+Filing
+------
+
+id
+    Open Civic Data-style id in the format ``ocd-cf-filing/{{uuid}}``
+
+filing_type
+    **optional**
+    Filing Type (jurisdiction-specific)
+
+filing_committee
+    Committee
+
+filing_date
+    Date (and possibly time) when filing was submitted.
+
+filing_coverage_begin_date
+    **optional**
+    Date (and possibly time) when filing period of coverage begins.
+
+filing_coverage_end_date
+    **optional**
+    Date (and possibly time) when filing period of coverage ends.
+
+from_organization
+    Regulator to which the Filing was submitted.
+
+filing_url
+    **optional**
+    Wish it wasn't optional.
+
+sources
+    **optional**
+    **repeated**
+    List of sources used in assembling this object. Has the following
+    properties:
+
+    url
+        URL of the resource.
+    note
+        **optional**
+        Description of what this source was used for.
+
+filing_relevant_election_date
+    Date of (nearest? next?) relevant election.
+
+filing_person
+    **optional**
+    Person responsible for the filing.
+
+Committee
+---------
+
+id
+    Open Civic Data-style id in the format ``ocd-cf-committee/{{uuid}}``
+
+name
+    Name of the Committee
+
+committee_type
+    Committee Type
+
+candidate orientations
+    **optional**
+    **repeated**
+    What posture the Committee takes towards specific Candidates. Committee type
+    can be imputed based on the number and nature of these orientations.
+
+officers
+    List of Persons who are the committee officers (maybe needs indication of
+    their ranks?)
+
+status
+    Current status of the Committee.
+
+purpose
```

Comment:
Sure, though I think we're gonna need to subclass Popolo Organizations rather than just use them as-is (mostly I think because of statuses, which are reasonably important in this context and go beyond date_dissolution and date_formation.